test/ansible/playbooks/subresources.yml

```diff
@@ -21,7 +21,7 @@
                 args:
                   - "/bin/sh"
                   - "-c"
-                  - "while true;do date;sleep 5;done"
+                  - "while true ; do echo '{{ log_message }}' ; sleep 5 ; done"
```

Comment:
Just our of curiosity ... the `log_message` here is from the `logMessage` of the CR?

Reply:
that's correct